content/questions/actions/question-123.md

```diff
@@ -0,0 +1,52 @@
+---
+question: "What must be added to `Checkout GitHub Action` if `my-org/my-repo` is a private repository differing from the one containing the current workflow?"
```

Comment:
All other snippets reference `my-org/my-private-repo`
```suggestion
question: "What must be added to `actions/checkout` if `my-org/my-private-repo` is a private repository differing from the one containing the current workflow?"
```

Reply:
Oops my apologies - just saw this. Thank you so much!